src/screens/GroupInfo/elements/ItemLink.js

```diff
@@ -69,7 +69,7 @@ const styles = StyleSheet.create({
     borderBottomRightRadius: 4
   },
   detail: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the detail background color should be verified for consistency with the rest of the application's color scheme.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->